ops/services/alerts/app_service_metrics/exceptions.tf

```diff
@@ -72,15 +72,20 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "first_error_in_a_week" {
   # - Collect all requests that were exceptions in the week preceeding today
   # - Do the same for today
   # - leftanti join the two result sets to return only results from today that were not found in the week preceeding today
+  # - note the first 'where' clause - the azurerm_monitor_scheduled_query_rules_alert resource doesn't allow intervals longer
+  #   than a day, and we only want to alert 1x/week. As a workaround, this `where` construction just checks if the current
+  #   DayOfYear (0-364) is evenly divisible by 7 before running the query.
   query = <<-QUERY
 requests
+${local.skip_on_weekends}
```

Comment:
Do you know if these two rules would conflict with each other at all? Not sure what would happen if a Saturday or Sunday was % 7 here. (I might also just be misunderstanding how this evenly divisible by 7 piece is working)

Reply:
Oh that's a good catch. I would assume whichever one came first would short circuit the other but we want to be careful of that for the alerts that run weekly. I'll look into this.

Reply:
I'm just going to hardcode this to a day and time per week, and remove the `skip_on_weekends` check, since I don't think these rules will play nice with each other.